pandas/tests/arrays/string_/test_string_arrow.py

```diff
@@ -3,14 +3,25 @@
 import numpy as np
 import pytest
 
-from pandas.core.arrays.string_arrow import ArrowStringArray
+from pandas.compat import pa_version_under1p0
 
-pa = pytest.importorskip("pyarrow", minversion="1.0.0")
+from pandas.core.arrays.string_arrow import (
+    ArrowStringArray,
+    ArrowStringDtype,
+)
 
 
+@pytest.mark.skipif(
+    pa_version_under1p0,
+    reason="pyarrow>=1.0.0 is required for PyArrow backed StringArray",
```

Comment:
should the numpy test still run?

Reply:
self.\_dtype = ArrowStringDtype() is defined in \_\_init__ before the input is checked. creating an instance ArrowStringDtype now raises an ImportError, tested in the test added.

Reply:
actually we checked pyarrow was availble before with `self._chk_pyarrow_available()` but this module was skipped with `pytest.importorskip("pyarrow", minversion="1.0.0")`